### DISCUSSION TITLE
gocryptfs on Android (prompt)

### DISCUSSION THREAD

**Discussion by misterhsp** [rfjakob/gocryptfs#744] [Q&A]
gocryptfs only works with the -nosyslog option on Android. The disadvantage is that the prompt is not returned. Continuing in the shell (termux) is not possible. Is there a way to change this or a trick to get the prompt again?

Thanks...

**Comment by lechner** [comment]
Hi,

I have never used the `-nosyslog` option, but I think you merely have a job control issue in your shell. In Bash, I would try something like `gocryptfs -nosyslog [...] >& /dev/null &`.

Kind regards
Felix Lechner

**Comment by misterhsp** [comment]
Thanks for the answer, unfortunately does not help. Prompt is not
returned. Maybe I should mention that it is Termux (shell).



Am 09.04.23 um 21:31 schrieb Felix Lechner:

**Comment by misterhsp** [comment]
It would be nice if there was a solution for this soon.

...
